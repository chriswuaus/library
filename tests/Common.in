COMMON 100001 1001001 10000
add member john stavrakakis 
common 100000 100000
add collection test.csv
common 100000 100001
add member heenaw
common 100000 10000000
common 100000 100000 1000001
common 100000 100001
common 100000
rent 100000 111111
rent 100000 111112
rent 100000 111113
rent 100000 111114
rent 100000 111115
relinquish 100000 111111
relinquish 100000 111112
relinquish 100000 111113
relinquish 100000 111114
relinquish 100000 111115
common 100000
rent 100001 111111
rent 100001 111113
rent 100001 111114
relinquish 100001 111111
relinquish 100001 111113
relinquish 100001 111114
common 100001
common 100001 100000
rent 100000 111111
relinquish 100000 111111
rent 100000 111111
relinquish 100000 111111
rent 100000 111111
relinquish 100000 111111
rent 100000 111111
relinquish 100000 111111
common 100000 100001
add member beanbabyth
rent 100002 111111
relinquish 100002 111111
common 100000 100001 100002
add member yeehaw
rent 100003 111120
relinquish 100003 111120
common 100000 100001 100002 100003
exit
