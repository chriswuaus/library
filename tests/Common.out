user: No members in system.

user: Success.

user: No books in system.

user: 17 books successfully added.

user: No such member in system.

user: Success.

user: No such member in system.

user: Duplicate members provided.

user: No common books.

user: No common books.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Iliad (Homer)
The Odyssey (Homer)

user: To Kill a Mockingbird (Harper Lee)
The Iliad (Homer)
The Odyssey (Homer)

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Iliad (Homer)
The Odyssey (Homer)

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Success.

user: Success.

user: Success.

user: No common books.

user: Ending Library process.
